### PR TITLE
Add additional host fields

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
This PR is to add additional host fields into ECS after RFC process https://github.com/kaiyan-sheng/ecs/blob/master/rfcs/text/0005-host-metric-fields.md is done.

### CODE REVIEW

**Comment by kaiyan-sheng** [comment]
@ebeahan When I tried to run make  it failed with no translation for `scaled_float`:
``` 
kaiyansheng@KaiyanMacBookPro:~/go/src/github.com/elastic/ecs (host_fields)$ make
build/ve/bin/python scripts/generator.py --strict --include ""
Loading schemas from local files
Running generator. ECS version 2.0.0-dev
build/ve/bin/python scripts/use-cases.py --stdout=true >> /dev/null
find code/go/ecs -name '*.go' -not -name 'doc.go' | xargs rm
cd scripts \
          && GO111MODULE=on go run cmd/gocodegen/gocodegen.go \
                -version=2.0.0-dev \
                -schema=../schemas \
                -out=../code/go/ecs
no translation for scaled_float (field cpu.usage)
exit status 1
make: *** [gocodegen] Error 1
```
Do you know how I can fix this? TIA!!

**Comment by ebeahan** [comment]
> Do you know how I can fix this? TIA!!

@kaiyan-sheng I've opened https://github.com/elastic/ecs/pull/1250 to fix this bug when `scaled_float` is used.

We didn't run into this with the `experimental` field definitions because the `go` generator only runs on against the  `schemas` directory.

**Comment by kaiyan-sheng** [comment]
@ebeahan Thank you! I updated the PR after successfully ran `make` command. Please let me know if these fields look ok to you!

**Review by ebeahan — COMMENTED** [review]
Thanks, @kaiyan-sheng, these changes look great!

Could we also pull out these changes from the experimental schema `host.*` file, [here](https://github.com/elastic/ecs/blob/master/experimental/schemas/host.yml#L4-L62), as well?

File: schemas/host.yml [hunk 1]
```diff
@@ -96,3 +96,69 @@
...
 
+    - name: cpu.usage
+      level: extended
+      type: scaled_float
+      scaling_factor: 1000
+      short: Percent CPU used, between 0 and 1.
+      beta: This field is currently considered beta.
+      description: >
+        Percent CPU used which is normalized by the number of CPU cores and it
+        ranges from 0 to 1. Scaling factor: 1000.
```
[the remark this answers is not in the document]
Comment (kaiyan-sheng) [reply schemas/host.yml]: Good point, fixed! Thank you!

**Comment by ebeahan** [comment]
@kaiyan-sheng @cyrille-leclerc I'd like to see these changes make it in before the upcoming FF for 1.9. I propose we move forward with introducing these changes, and we can open a separate PR after FF to remove the `beta` attribute if we determine the host metrics fields can be considered GA.

Any objections?

**Comment by kaiyan-sheng** [comment]
@ebeahan Sounds good to me. Thank you! I will fix the conflict right now.

**Review by ebeahan — APPROVED** [review]
LGTM 👍

**Pull request merged**